EnglishFilter/sections/specific.txt

```diff
@@ -254,6 +254,10 @@ feetporno.com,familyporn.tv,freehardcore.com,webvoyeur.com,moviesand.com,momvids
 !
 !
 !
+uploader.*##a[href*="https://shopcart4me.com"]
```

Comment:
Why did you use `*`? Please provide some examples.
Why not `uploader.link##a[href^="https://shopcart4me.com"]`?

Reply:
I didn't look up but I think they have more domains.